pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -301,6 +311,40 @@ private bool ValidateImportResultsArgs()
             return true;
         }
 
+        public static readonly Argument ARG_LIMIT_NOISE = new DocArgument(@"limit-noise", NUM_VALUE,
+            (c, p) => c.LimitNoise = p.ValueDouble);
+
+        public static readonly Argument ARG_CHROMATOGRAM_DISCARD = new DocArgument(@"chromatogram-discard",
+            (c, p) => c.ChromatogramsDiscard = true );
+
+        private static readonly ArgumentGroup GROUP_MINIMIZE_RESULTS = new ArgumentGroup(() => CommandArgUsage.CommandArgs_GROUP_MINIMIZE_RESULTS_Minimizing_results_file_size, false,
+            ARG_LIMIT_NOISE, ARG_CHROMATOGRAM_DISCARD)
+        {
+            Validate = c => c.ValidateMinimizeResultsArgs()
+        };
+        
+        private bool ValidateMinimizeResultsArgs()
+        {
+            if (Minimizing && !(_seenArguments.Contains(ARG_SAVE) || _seenArguments.Contains(ARG_OUT)))
+            {
+                // Has minimize argument(s), but no --save or --out command
```

Comment:
I think just error and stop. It is better to error and fail to do anything than continue and do something undesirable or unintended.